### PR TITLE
STYLE: remove duplicated itkMath.h headers

### PR DESCRIPTION
**Pull request by issakomi**
17 files had `#include "itkMath.h"` twice and one file 3 times.

### CODE REVIEW

File: Modules/Filtering/FastMarching/include/itkFastMarchingImageFilter.hxx [hunk 1]
```diff
@@ -20,9 +20,7 @@
 
 #include "itkImageRegionIterator.h"
 #include "itkNumericTraits.h"
-#include "itkMath.h"
 #include <algorithm>
-#include "itkMath.h"
```
Comment (issakomi) [inline Modules/Filtering/FastMarching/include/itkFastMarchingImageFilter.hxx:25]: Already included in itkFastMarchingImageFilter.h

**Review by dzenanz — APPROVED** [review]

**Comment by N-Dekker** [comment]
FYI Such duplicate `#include`'s typically seem to be introduced by git merges. So it's not like people intentionally added the same `#include` twice.

**Review by N-Dekker — APPROVED** [review]
Thanks Mihail! Cool!

**Pull request merged**